scripts/assets/test_file_exemptions.textproto

```diff
@@ -351,6 +351,8 @@ exempted_file_path: "app/src/main/java/org/oppia/android/app/settings/profile/Pr
 exempted_file_path: "app/src/main/java/org/oppia/android/app/settings/profile/ProfileRenameActivityPresenter.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/settings/profile/ProfileRenameViewModel.kt"
 exempted_file_path: "app/src/main/java/org/oppia/android/app/settings/profile/ProfileResetPinActivityPresenter.kt"
+exempted_file_path: "app/src/main/java/org/oppia/android/app/settings/profile/ProfileResetPinFragment.kt"
```

Comment:
Remove this and add `ProfileResetPinFragmentTest` file and transfer tests from `ProfileResetPinActivityTest` file to this as required making sure that it resonates with the code written in `ProfileResetPinFragment` and `ProfileResetPinFragmentPresenter`

Reply:
done